src/pages/index.js

```diff
@@ -64,25 +125,75 @@ const popupAddCardForm = new PopupWithForm({
 });
 popupAddCardForm.setEventListeners();
 
+const profileAvatarForm = new PopupWithForm({
+  popupSelector: "#avatar-modal",
+  handleFormSubmit: handleProfileAvatarSubmit,
+});
+profileAvatarForm.setEventListeners();
+
+const deleteCardConfirm = new PopupWithConfirmation({
+  popupSelector: "#delete-card-modal",
+});
+deleteCardConfirm.setEventListeners();
+
+const previewImage = new PopupWithImage({
+  popupSelector: "#modal__preview-image",
+});
+previewImage._setEventListeners();
+
 const editFormValidator = new FormValidator(config, profileEditForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```